### PR TITLE
Added 63. Unique Paths 2 solution in C++

### PR DESCRIPTION
**Pull request by pranshta**
### What have you Changed(must)
Added solution to problem " 63. Unique Paths 2  " (https://leetcode.com/problems/unique-paths-ii/)

Write here what you have changed in the codebase
Added a solution for problem number 63 of Leetcode

### Issue no.(must) - #722 

Pr will be close and marked as spam. If issue number not found or issue was assigned to someone else. 
Marking as spam can block your account from HacktoberFest.

### Self Check(Tick After Making pull Request)

- [x] This issue was assigned to me if not it will be marked as spam.
- [x] My file is in the proper folder
- [x] I am following clean code and Documentation.
- [x] I have added the title and what program will do first in the file
- [x] I have named the file as (Problem #). (Problem Title(in Camel Case)).(extension)
      ex. 2. AddToNumbers.py

### CODE REVIEW

**Review by Yashica7 — CHANGES_REQUESTED** [review]
Add what your code does

File: C++/UniquePaths||.cpp [hunk 1]
```diff
@@ -0,0 +1,56 @@
+class Solution {
```
Comment (pranshta) [inline C++/UniquePaths||.cpp:1]: /*
Problem Satement:-
A robot is located at the top-left corner of a m x n grid 
The robot can only move either down or right at any point in time. The robot is trying to reach the bottom-right corner of the grid (marked 'Finish' in the diagram below).
Now consider if some obstacles are added to the grids. How many unique paths would there be?
An obstacle and empty space is marked as 1 and 0 respectively in the grid.
The solution uses Dynamic approach to find the total number of unique paths for the robot to reach end point (n-1,m-1) from start point(0,0)
*/

Comment (pranshta) [inline C++/UniquePaths||.cpp:1]: /*
Problem Satement:-
A robot is located at the top-left corner of a m x n grid 
The robot can only move either down or right at any point in time. The robot is trying to reach the bottom-right corner of the grid (marked 'Finish' in the diagram below).
Now consider if some obstacles are added to the grids. How many unique paths would there be?
An obstacle and empty space is marked as 1 and 0 respectively in the grid.
The solution uses Dynamic approach to find the total number of unique paths for the robot to reach end point (n-1,m-1) from start point(0,0)
*/

**Comment by pranshta** [comment]
@Yashica7 I am unable to edit the file in this PR ...had created another PR #893 ,but it was closed

**Comment by SSKale1** [comment]
We're closing this PR too now. You make a new PR with the proper comments.

**Pull request closed without merging**